Loop/Managers/WatchDataManager.swift

```diff
@@ -423,7 +423,9 @@ extension WatchDataManager: WCSessionDelegate {
 
                 // Prevent re-sending these updated settings back to the watch
                 lastSentSettings = settings
-                deviceManager.loopManager.settings = settings
```

Comment:
This is the kind of write that would actually be ok. So forcing a setter func with a closure is a step backwards here, but I think preventing direct writes is probably still a good idea.